### DISCUSSION TITLE
希望打卡集也能支持标签功能

### DISCUSSION THREAD

**Discussion by musoulee** [WatsonYao/nian_app#118] [Q&A] [answered]
最近突发奇想，用打卡集记录自己每次的💩时间和地点，如果地点能用标签分类就好了，这样我能知道自己最偏爱什么地方。
好小众的需求......不知道有没有人跟我一样

**Comment by WatsonYao** [comment]
是指某个打卡记本里面，支持写进展的时候，添加标签？

**Reply by musoulee** [reply]
对滴！其实就是希望打卡集能多一些维度去展示，比如地点、星期几、时间段等等，这些维度只是我自己想的，具体可能因人而异。所以就想着可不可以做成标签的形式，这样自定义程度高一点。

**Reply by WatsonYao** [reply, marked as answer]
已在后续版本安排，打卡本进展支持标签。

**Reply by lilac317** [reply]
既然打卡也能加标签，那能不能清单也加标签，可以查看这个清单项属于什么项目，还可以用时间标签来查看这个清单项是什么时候应该完成的任务。

**Reply by WatsonYao** [reply]
清单之前没有考虑和标签以及日期挂钩，所以不打算提供更多功能了。可能会考虑重新弄一个待办事项之类的。
